All repository set tests are failing with KeyError: 'rhva6'

```
tests/foreman/api/test_repository_set.py:81: in test_positive_reposet_disable
    name=REPOSET['rhva6'],
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <robottelo.host_info.SatVersionDependentValues object at 0x7ff0c42bb490>
item = 'rhva6'

    def __getitem__(self, item):
        """
            Return value dependent on Satellite version
            :param item: str
            :return: respective Satellite version values
            """
    
        sat_version = get_host_sat_version()
        try:
            return self._versioned_values[sat_version][item]
        except KeyError:
>           return self._common[item]
E           KeyError: 'rhva6'

robottelo/host_info.py:110: KeyError

```

--- a/robottelo/host_info.py
+++ b/robottelo/host_info.py
@@ -29,11 +29,7 @@ def get_host_os_version():
     return 'Not Available'
 
 
-_SAT_6_2_VERSION_COMMAND = (
-    u'grep "SATELLITE_SHORT_VERSION" '
-    u'/opt/theforeman/tfm/root/usr/share/gems/gems/foreman_theme_satellite-0'
-    u'.1.25/lib/foreman_theme_satellite/version.rb'
-)
+_SAT_6_2_VERSION_COMMAND = u'rpm -q satellite'
 
 _SAT_6_1_VERSION_COMMAND = (
     u'grep "VERSION" /usr/share/foreman/lib/satellite/version.rb'

--- a/tests/robottelo/test_host_info.py
+++ b/tests/robottelo/test_host_info.py
@@ -156,7 +156,7 @@ class GetHostSatVersionTestCase(TestCase):
     def test_sat_6_dot_2(self):
         """Check if can parse major 6.2.x versions"""
         self.assert_sat_version(
-            u'  SATELLITE_SHORT_VERSION = "6.2.7"',
+            u'satellite-6.2.0-21.1.el7sat.noarch',
             u'6.2'
         )
 

